pkg/machinery/config/types/v1alpha1/v1alpha1_features.go

```diff
@@ -38,3 +38,28 @@ func (f *FeaturesConfig) ApidCheckExtKeyUsageEnabled() bool {
 func (f *FeaturesConfig) DiskQuotaSupportEnabled() bool {
 	return pointer.SafeDeref(f.DiskQuotaSupport)
 }
+
+// APIServerBalancer implements config.Features interface.
+func (f *FeaturesConfig) APIServerBalancer() config.APIServerBalancer {
+	if f.APIServerBalancerSupport == nil {
+		return &APIServerBalancer{}
+	}
+
+	return f.APIServerBalancerSupport
+}
+
+const defaultAPIBalancerPort = 7445
```

Comment:
maybe a much higher port in the 50000 range since that's what talos usually uses

Reply:
@smira asked for a lower than 10000 port. We agreed on 7445 during standup call.